src/main.rs

```diff
@@ -88,20 +90,21 @@ impl Deno {
         Deno { ptr: ptr }
     }
 
-    fn execute(&self, js_filename: &str, js_source: &str) -> bool {
+    fn execute(&mut self, js_filename: &str, js_source: &str) -> Result<(), DenoException> {
         let filename = CString::new(js_filename).unwrap();
```

Comment:
I would like to suggest to introduce a real error type (an enum which encapsulates all the possible errors in the function), and get rid of `unwrap`s.

Reply:
Can do that tomorrow. Meanwhile please comment on https://github.com/robbym/deno/pull/1